index.html

```diff
@@ -0,0 +1,180 @@
+
+    <!-- Social Media Share Links: LinkedIn, Facebook, Etc. -->
+    <meta
+      name="image"
+      property="og:image"
+      content="https://github.com/CodeMeKathy/cmk-dev-happiness/blob/master/assets/GitHub%20Social%20Preview%20Small.png?raw=true"
+    />
+
+    <!-- WhatsApp -->
+    <meta
+      name="image"
+      property="image"
+      content="https://github.com/CodeMeKathy/cmk-dev-happiness/blob/master/assets/GitHub%20Soc   ial%20Preview%20Small.png?raw=true"
+    />
+
+    <meta
+      property="og:title"
+      content="Kathy Lambert | Developer Extraordinaire"
+    />
+    <meta property="og:url" content="https://kathylambert.me" />
+    <meta
+      property="og:description"
+      content="Kathy Lambert | Developer Extraordinaire Personal Portfolio"
+    />
+    <meta
+      name="”description”"
+      content="”Kathy"
+      Lambert
+      |
+      Developer
+      Extraordinaire
+      Personal
+      Portfolio”
+    />
+    <meta property="og:type" content="website" />
+    <meta property="og:locale" content="en_US" />
+
+    <!-- Author Information -->
+    <meta
+      name="author"
+      content="Kathy Lambert Personal Portfolio"
+      href="https://kathylambert.me"
+    />
+    <meta
+      name="author"
+      content="Kathy Lambert GitHub Portfolio"
+      href="https://github.com/CodeMeKathy"
+    />
+
+    <!-- Google Fonts -->
+    <link
+      href="https://fonts.googleapis.com/css?family=Raleway"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato:100,300,400,700,900"
+      rel="stylesheet"
+    />
+    <!-- Favicon -->
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/apple-touch-icon.png?v=eEG6YMEev5"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="/favicon-32x32.png?v=eEG6YMEev5"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="/favicon-16x16.png?v=eEG6YMEev5"
+    />
+    <link rel="manifest" href="/site.webmanifest?v=eEG6YMEev5" />
+    <link
+      rel="mask-icon"
+      href="/safari-pinned-tab.svg?v=eEG6YMEev5"
+      color="#40c0cb"
+    />
+    <link rel="shortcut icon" href="/favicon.ico?v=eEG6YMEev5" />
+    <meta name="apple-mobile-web-app-title" content="Kathy Lambert" />
+    <meta name="application-name" content="Kathy Lambert" />
+    <meta name="msapplication-TileColor" content="#40c0cb" />
+    <meta name="theme-color" content="#40c0cb" />
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="./style.css" />
+
+    <title>ResumeBuilder</title>
+  </head>
+  <body>
+  <header></header>
+  <main>
+    <h1>Resume Builder</h1>
+    <div class="container">
+      <form id='resumeInputForm'>
```

Comment:
This form is very large and doesn't yet have all the inputs for the full resume. It might be worth looking into a way to separate different forms into different pages, modularizing the form somehow, or using React to make it modular.